account_banking_sepa_direct_debit/models/mail_compose_message.py

```diff
@@ -10,7 +11,7 @@ class MailComposeMessage(models.TransientModel):
     def _action_send_mail(self, auto_commit=False):
         for wizard in self:
             if self.env.context.get("is_sent"):
-                self.env[wizard.model].sudo().browse(wizard.res_id).is_sent = True
-        return super(MailComposeMessage, self)._action_send_mail(
-            auto_commit=auto_commit
-        )
+                self.env[wizard.model].sudo().browse(
+                    json.loads(wizard.res_ids)
```

Comment:
Why do you need to convert it to JSON?

Reply:
It is returning list in string format that's why